x-pack/test/api_integration/apis/security/basic_login.js

```diff
@@ -145,6 +145,7 @@ export default function ({ getService }) {
         'authentication_realm',
         'lookup_realm',
         'authentication_provider',
+        'authentication_type',
       ]);
       expect(apiResponse.body.username).to.be(validUsername);
     });
```

Comment:
I'm looking at this for the first time; do you think we should be verifying the `authentication_realm` , `authentication_provider`, and `authentication_type` here too? I know it's a bit more than the scope of this PR but the test as written seems incomplete.